Documentation/manualtest-guide.md

```diff
@@ -81,20 +60,21 @@ Requirements:
 
 * Requires a Windows machine running the Bridge
 * The Windows machine should be on a Windows domain 
-* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/>
+* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/> 
+  Download the tool from https://technet.microsoft.com/en-us/sysinternals/psexec.aspx <br />
   `psexec -s -h <command>` will start a command as an elevated LOCAL SYSTEM user
 
```

Comment:
Is Step 1 below not sufficient? 


> 1. Ensure the Bridge is running on a Windows machine and as LOCAL SYSTEM with `psexec -s -h ${GitWcfRoot}\startBridge.cmd -allowRemote -remoteAddresses:*`

I preferred to allow startBridge.cmd do the hard work rather than passing in the BridgeResourceFolder